### PR TITLE
[Tech] Restore missing css variables

### PR DESCRIPTION
**Pull request by dawidgarus**
Recent PR removed some css variables which were used:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1064
I changed code to not to used them in some places and restored variables that were still missing

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Comment by 8Bitz0** [comment]
I assumed because the variables had a comment saying that they should be avoided and did not follow the same naming, they should be removed altogether.

If something needed to use the variables, shouldn't they be switched to a new one? (Or get a new variable?)

Or am I missing something else here?

**Comment by dawidgarus** [comment]
> I assumed because the variables had a comment saying that they should be avoided and did not follow the same naming, they should be removed altogether.

They should if they are not used. But some of them were. It's probably because changes were made in main branch before the branch was merged.

They only way to ensure things like this won't happen is to manually pull changes from main branch and test before merging PR into main branch. Which ofc may be problematic if contributor and maintainer don't work on the same schedule.

> If something needed to use the variables, shouldn't they be switched to a new one? (Or get a new variable?)

Some of them were changed to new ones. Some of them I kept for now because there weren't any variable with a name that would match yhe intended purpose. I didn't want to dive to deep into it here, only fix what was broken.

**Review by flavioislima — APPROVED** [review]
Good catch. I thought they were replaced in the code but I believe not all of them were.

**Pull request merged**
